public/static/docs/command-reference/update.md

```diff
@@ -27,20 +27,32 @@ Note that import stages are considered always locked, meaning that if you run
 update them.
 
 `dvc update` will not have an effect on import stages that are fixed to a commit
-hash (`rev` field in the DVC-file). Please refer to
-[Fixed revisions & re-importing](/doc/command-reference/import#example-fixed-revisions-re-importing)
-for more details.
+hash (`rev` field in the DVC-file). To update the imported artifacts to a
+certain revision, `--rev` with specified revision can be used.
+
+```dvc
+dvc update --rev master
+```
 
 ## Options
 
+- `--rev` - specific
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  (such as a branch name, a tag, or a commit hash) of the repository to update
+  the file or directory from (also starts tracking the given revision).
+
+  > Note that this adds or updates a `rev` field in the DVC-file that fixes it
+  > to this revision (and updates `rev_lock` in the DVC-file). This can have an
+  > impact on the behavior of `dvc update` later.
+
 - `-h`, `--help` - prints the usage/help message, and exit.
 
 - `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
   problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Examples
+## Example: Updating imported artifacts
```

Comment:
Just
```suggestion
## Example
```
should be enough here. "Updating imported artifacts" is what this command is about anyway.